specification/securityinsights/resource-manager/readme.md

```diff
@@ -53,11 +53,21 @@ These settings apply only when `--tag=package-2021-03-preview-only` is specified
 input-file:
 - Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json
 - Microsoft.SecurityInsights/preview/2021-03-01-preview/operations.json
+- Microsoft.SecurityInsights/preview/2021-03-01-preview/EntityQueryTemplates.json
+- Microsoft.SecurityInsights/preview/2021-03-01-preview/EntityQueries.json
 directive:
   - suppress: R4017
     from: Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json
     where: $.definitions.Settings
-    reason: The Setting does not support list by subscription. It's not a top-level resource. To get the Watchlist, we should have a subscription as well as a resource group and Log Analytics workspace. 
+    reason: The Setting does not support list by subscription. It's not a top-level resource. To get the Watchlist, we should have a subscription as well as a resource group and Log Analytics workspace.
+  - suppress: R4017
```

Comment:
Do we need the suppress for every file? can we mention it once for all resources?

Reply:
this is a good question; I'm relying on @yuvalshilo implementation